frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -102,10 +106,10 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
   };
 
   return (
-    <Tooltip
-      role="none"
-      data-testid="model-status-tooltip"
-      content={
+    <Popover
```

Comment:
It's possible to make PF popovers trigger on `hover`, do we know if UX wants to move to on click? or keep hover like a tooltip?

Reply:
The figma makes it seem that they want a click. One of the issues of this bug was with the hover, when you went to select the text, moving the mouse away from the status caused the message to disappear. I'm unsure if that would be an issue with the popover as well though